app/serializers/user_serializer.rb

```diff
@@ -8,7 +8,7 @@ class UserSerializer < ActiveModel::Serializer
 
   attributes :services, :account, :profile, :va_profile, :veteran_status,
              :in_progress_forms, :prefills_available, :vet360_contact_information,
-             :session
+             :session, :onboarding
```

Comment:
This *does* update the existing data contract, but creates no breaking changes for the client. 

Once we have arrived at a v1 of this feature, we will formally update the Swagger API documentation.